apps/weather/settings.js

```diff
@@ -1,31 +1,87 @@
-(function(back) {
-  const storage = require('Storage');
-  let settings = storage.readJSON('weather.json', 1) || {};
+(back) => {
```

Comment:
As per the checker, this function needs to be wrapped with parentheses and exclude the finishing semi-colon:

```
((back) => {
....
})
```